junit5/src/main/java/com/linecorp/armeria/testing/junit5/client/AssertThat.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.testing.junit5.client;
+
+import static java.util.Objects.requireNonNull;
+
+abstract class AssertThat<T, U> {
+    private final T actual;
+    private final U back;
+
+    AssertThat(T actual, U back) {
+        requireNonNull(actual, "actual");
+        requireNonNull(back, "back");
+        this.actual = actual;
+        this.back = back;
+    }
+
+    T actual() {
+        return actual;
+    }
+
+    U back() {
+        return back;
+    }
```

Comment:
Question: Where `back` term came from?

Reply:
When I named it, I intended for the name to convey the idea of returning or coming back. 😅 Upon reconsideration, it was an inappropriate name.

I'd like to hear your opinion before choosing a more appropriate name. I think `AssertThat` class is essentially the same as a Tuple or Pair. What do you think would be better: changing the `AssertThat` class to `Pair`, or just changing the `back` term to a more appropriate name?

Reply:
If it is the assertion value, I prefer using `expected` because the name is commonly used in testing libraries.

Reply:
I realized that it is used a bit differently than I expected. Let me give you feedback after reading all the changes.